src/c/cancel-project-modal-content.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * window.c.cancelProjectModalContent component
+ * Render cancel project modal
+ *
+ */
+import m from 'mithril';
+import models from '../models';
+import postgrest from 'mithril-postgrest';
+
+const cancelProjectModalContent = {
+    controller(args) {
+        let l = m.prop(false),
+            deleteSuccess = m.prop(false),
+            checkError = m.prop(false),
+            check = m.prop('');
+
+        const showNextModal = () => {
+            if (check() === 'cancelar-projeto'){
+              args.displayModal.toggle();
+              document.getElementById('send-message').style.display = 'block';
```

Comment:
Yes, we can port redactor to the API later, but this would take too much effort right now.